### PR TITLE
Fix wrong reference website for docs.

### PR DESCRIPTION
**Pull request by yanggangtony**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix wrong reference website for docs.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ labels: kind/question
 
 <!-- Please use this template while asking a new question. Thanks!
 Please make sure you have read the FAQ and searched the issue list.
-- FAQ: https://github.com/karmada-io/karmada/blob/master/docs/frequently-asked-questions.md
+- FAQ: https://github.com/karmada-io/website/blob/main/docs/faq/faq.md
```
Comment (RainbowMango) [inline .github/ISSUE_TEMPLATE/question.md:10]:
```suggestion
- FAQ: https://karmada.io/docs/faq
```
How about link to the website?

Comment (RainbowMango) [reply .github/ISSUE_TEMPLATE/question.md:10]: Same as below.
I would tend to link to the website because the content in a repo might change over time.

Comment (yanggangtony) [reply .github/ISSUE_TEMPLATE/question.md]: Rebased.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve
Thanks.

**Pull request merged**